### PR TITLE
include current tasks on work overview

### LINKED ISSUE #682: As a contributor I want to see a list/overview of open tasks for a work on the work overview page

### PR DESCRIPTION
**Pull request by goose-life**
closes #682

image

### CODE REVIEW

File: indigo_app/templates/indigo_api/_task_list.html [hunk 1]
```diff
@@ -6,10 +6,14 @@
     <tr>
       {% block table-header %}
       <th scope="col" colspan="2">Task</th>
-      {% if place %}
-        <th scope="col">Place</th>
+      {% if not work_overview %}
```
Comment (longhotsummer) [inline indigo_app/templates/indigo_api/_task_list.html]: This breaks the concept of encapsulation, that this template should only need to know about its local state. `work_overview` implies it has broader knowledge, that it understands that when it's being used in a particular context, then certain things should happen differently.

The previous `place` indicator inverts this: the including context is telling the template to behave in a certain way. This preserves encapsulation.

Instead, I suggest you keep `place` and either just leave 'Work/Document' as it is (actually I think "Work or Document" is better), or add another variable to control the display of that column.

File: indigo_app/templates/indigo_api/work_overview.html [hunk 2]
```diff
@@ -190,4 +190,11 @@ <h3 class="mb-0">{{ work.metrics.p_breadth_complete|floatformat:0 }}%</h3>
   {% include 'indigo_api/_work_points_in_time.html' with work=work editable=False %}
   {% endblock %}
 
+  {% block active_tasks %}
+  <div class="h4 mb-3 mt-5">Current tasks</div>
```
Comment (longhotsummer) [inline indigo_app/templates/indigo_api/work_overview.html]:
```suggestion
  <div class="h4 mb-3 mt-5">Open tasks</div>
```

File: indigo_app/templates/indigo_api/_task_list.html (hunk 1 above)
Comment (goose-life) [reply indigo_app/templates/indigo_api/_task_list.html]: I would like to not display the work when on the work overview, it feels silly to include it.
When you say 'add another variable to control the display of that column', would it be fine to just call it something else, like `dont_display_works`? I don't want to make it `display_works` for all other instances, the default should be to display them.

Comment (goose-life) [reply indigo_app/templates/indigo_api/_task_list.html]: something like
```
  {% if hide_works %}
    <th scope="col">Document</th>
  {% else %}
    <th scope="col">Work or Document</th>
  {% endif %}
```

Comment (longhotsummer) [reply indigo_app/templates/indigo_api/_task_list.html]: Yes, absolutely. It's exactly the same concept as 'show/don't show the place'.

**Pull request merged**